documents/policies/Competition Requirements.md

```diff
@@ -0,0 +1,145 @@
+   2. Capitalization of the competition name must be appropriate. The first letter of each word in the competition’s name should be capitalized and subsequent letters within words should be lowercase, except articles and prepositions, which should be always lowercase. Acronyms should be capitalized.
+   3. The name must end with the year of the last day of the competition.
+   4. The competition name must be in Roman-readable letters and may only contain characters from A-Z, a-z, 0-9, or similar characters. Only '.:-& are permitted as special characters.
+   5. The competition name must be 50 characters or less, including whitespaces.
+   6. If the competition is named after anyone or anything, they must be persons or things lacking any controversial meaning worldwide.
+   7. Names should contain the location of the competition.
+   8. Competitions may not be named after a living cuber, competition organizer, or any competition staff.
+   9. Names must not contain possibly offensive parts or memes.
+   10. The name cannot contain website domains or anything susceptible to be interpreted as a link, hashtag or in general an electronic identifier.
+   11. The name cannot contain any trademarks unless the organization team is formally authorized, in writing, to use such names.
+   12. Names should not contain any unnecessary parts in the name, like "competition".
+   13. The competition nickname and ID must resemble the competition name.
+       1. The competition nickname and ID should only be modified if the competition name is too long (more than 32 characters). Exception: groups of competitions may have modified nicknames and IDs to ensure consistency with past and future competitions.
+    14. Abbreviations and acronyms should be avoided unless the competition name is too long (more than 32 characters).
+2. Location(s)
+   1. A location consists of the country, city, and venue.
+   2. The location should also include subregions of the country that are common to include with addresses (e.g. the state a U.S. city is located in).
+   3. Competitions will be accepted if there are no other competitions within 100 km driving distance and less than 19 days away.
+      1. Otherwise, the WCA Competition Announcement Team should review the circumstances of the proposed competition. The WCA Delegate requesting approval for such a competition must provide further information on why the competition should still be accepted. Possible arguments that support such a competition being accepted include:
+         1. The two competitions don’t have events in common.
+         2. The two competitions take place in different countries.
+   4. 3x3x3 Fewest Moves simultaneous competitions
+      1. 3x3x3 Fewest Moves simultaneous competitions are WCA competitions having only the 3x3x3 Fewest Moves event and taking place simultaneously in different locations.
+         1. If any location of such competitions is closer than 100 km to any other competition, then
```

Comment:
```suggestion
         1. Like for a regular competition, the deadline for adding a new location to a 3x3x3 Fewest Moves simultaneous competition is 28 days. 
         2. If any location of such competitions is closer than 100 km to any other competition, then
```

Reply:
Maybe `Any location must be announced at least 28 days (four weeks) before 00:00 UTC of the start date of the competition on the WCA website.` is better for consistency?

Reply:
I edited the original suggestion according to my comment and I'm committing it!